src/components/bookForm.js

```diff
@@ -1,24 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Book from './books';
+import { addbook, removebook } from '../redux/Books/book';
 
-const BooksForm = () => (
-  <div>
-    <h2>Add new book</h2>
-    <form>
-      <input
-        type="text"
-        name="title"
-        placeholder="Add title"
-      />
-      <input
-        type="text"
-        name="author"
-        placeholder="Add author"
-      />
-      <button type="submit" className="submit-btn">
-        Add book
-      </button>
-    </form>
-  </div>
-);
+function Books() {
+  const [name, setName] = useState('');
+  const [btitle, setTitle] = useState('');
+  const books = useSelector((state) => state.books.books);
```

Comment:
As you make some changes on your files, remember to test it after doing changes and pushing to Github.

Seems that the changes you made wroke your app. Now there's an error in the console and nothing appearing on the screen. 
- Kindly fix this error to test the `remove` method

![image](https://github.com/Daniel-Kigozi/react-book2/assets/93630700/c172c5bf-2bc8-41ce-9238-e76059442a0c)